File: 프로그래머스/lv0/120887. k의 개수/k의 개수.py
def solution(i, j, k):
    #i(숫자) 부터 j(숫자)까지 범위가 주어질때 k라는 수가 얼마나 나오는지
    #이거 무조건 join으로 푸는 문제구나 와 나쁘다 
    new_list =[]
    cnt=0
    for s in range(i,j+1):
        new_list.append(str(s))
    a= "".join(new_list)
    for h in a:
        if int(h)==k:
            cnt+=1
            
    return cnt

File: 프로그래머스/0/120887. k의 개수/k의 개수.js
function solution(i, j, k) {
    var answer = 0;
    let num_list = []
    for(let s =i;s<=j;s++){
        num_list.push(s)
    }
  num_list = num_list.join("").split("")
  num_list =   num_list.filter((e)=> e==k)
    return num_list.length;
}